views/admin-meta-box-woocommerce-order.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * WordPress admin meta box WooCommercer order Pronamic Pay
+ *
+ * @author    Pronamic <info@pronamic.eu>
+ * @copyright 2005-2023 Pronamic
+ * @license   GPL-3.0-or-later
+ * @package   Pronamic\WordPress\Pay
+ */
+
+$payment_id = (int) $order->get_meta( '_pronamic_payment_id' );
+
+$payment = get_pronamic_payment( $payment_id );
+
+?>
+<style>
+	#woocommerce-order-pronamic-pay .inside {
+		margin: 0;
+		padding: 10px;
+	}
+
+	#woocommerce-order-pronamic-pay th {
+		text-align: left;
+	}
+</style>
+
+<?php if ( null === $payment ) : ?>
+
+	<?php esc_html_e( 'No Pronamic Pay payment found for this WooCommerce order.', 'pronamic_ideal' ); ?>
+
+<?php else : ?>
+
+	<table>
+		<tbody>
+			<tr>
+				<th scope="row">
+					<?php esc_html_e( 'ID' ); ?>
+				</th>
+				<td>
+					<?php edit_post_link( $payment->get_id(), '', '', $payment->get_id() ); ?>
+				</td>
+			</tr>
+			<tr>
+				<th scope="row">
+					<?php esc_html_e( 'Date', 'pronamic_ideal' ); ?>
+				</th>
+				<td>
+					<?php echo esc_html( $payment->date->format_i18n() ); ?>
+				</td>
+			</tr>
+			<tr>
+				<th scope="row">
+					<?php esc_html_e( 'Amount', 'pronamic_ideal' ); ?>
+				</th>
+				<td>
+					<?php echo esc_html( $payment->get_total_amount()->format_i18n() ); ?>
+				</td>
+			</tr>
+			<tr>
```

Comment:
Also include payment status in the meta box?

```suggestion
			<tr>
				<th scope="row">
					<?php esc_html_e( 'Status', 'pronamic_ideal' ); ?>
				</th>
				<td>
					<?php echo esc_html( $payment->get_status_label() ); ?>
				</td>
			</tr>
			<tr>
```